BUILD.bazel

```diff
@@ -2,7 +2,7 @@
 
 load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library", "cc_proto_library")
 load("@rules_java//java:defs.bzl", "java_lite_proto_library", "java_proto_library")
-load("@rules_pkg//:mappings.bzl", "pkg_files", "strip_prefix")
+load("@rules_pkg//:pkg:mappings.bzl", "pkg_files", "strip_prefix")
```

Comment:
Should this be `//pkg:mappings.bzl` instead of `//:pkg:mappings.bzl`?

Reply:
yep - bad sed-fu - ill update now